### ISSUE TITLE
Add Current Status option 

### ISSUE THREAD

**Issue by UsuallyLogical** [scs-ben/tcec-gui#146]
Currently, a visitor to TCEC has to figure out what's going on by asking in Chat, and maybe getting an answer that will roll off within seconds.

How about adding (something like) a "Current Status" menu item that the user could click on to find out all the pertinent information. For example, and the time of this posting, it might say:

"This is TCEC season 13. It has been determined that the superfinal will be a match between Stockfish and Komodo, and it will begin on Oct 22 at 20:00 CEST. In the meantime we are hosting a knockout tournament between this season's engines."

In other words, you have a place that can answer many questions and make announcements, without having to annoy the chat people who have probably answered the same questions many times.

**Comment by arunsathya1309** [comment]
Info box is supposed to be for that
